### PR TITLE
Widget space automation

### PR DESCRIPTION
**Pull request by adamweeks**
Adds automation tests for widget space

### CODE REVIEW

File: test/journeys/specs/oneOnOne/basic.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    `spark:rooms_write`,
+    `spark:memberships_read`,
+    `spark:memberships_write`,
+    `spark:messages_read`,
+    `spark:messages_write`,
+    `spark:teams_read`,
+    `spark:teams_write`,
+    `spark:team_memberships_read`,
+    `spark:team_memberships_write`,
+    `spark:kms`
```
Comment (bzang) [inline test/journeys/specs/oneOnOne/basic.js:24]: Will need to switch this over to `spark:all` whenever that becomes available.

**Review by bzang — APPROVED** [review]
Looks good!

File: test/journeys/specs/oneOnOne/basic.js (hunk 1 above)
Comment (adamweeks) [reply test/journeys/specs/oneOnOne/basic.js:24]: We can probably remove the SCIM scopes too.

**Comment by adamweeks** [comment]
@bzang I don't have write access to this so feel free to merge

**Pull request merged**